generators/server/templates/src/main/java/package/security/jwt/TokenProvider.java.ejs

```diff
@@ -100,8 +100,9 @@ public class TokenProvider {
     }
 
     public Authentication getAuthentication(String token) {
-        Claims claims = Jwts.parser()
+        Claims claims = Jwts.parserBuilder()
```

Comment:
Ideally, you want to reuse the parser `parserBuilder()...build()` when possible. (and only incur the cost of construction once)
Maybe stick it in the PostConstruct above?

Reply:
You're right @bdemers, I guess I was too focused in the deprecation warning and absolutely not in the performance.